content/expressions/control-structures.md

```diff
@@ -122,6 +122,20 @@ actor Main
 
 This will give __x__ the value "Sarah" as it is the last name in our list. If our loop has 0 iterations, then the value of its `else` block will be the value of __x__. Or if there is no `else` block, the value will be `None`.
 
+You can also avoid needing `None` at all by providing a __default value__ for when the loop has __0 iterations__ by providing an `else` block.
+
+```pony
+  actor Main
+  new create(env: Env) =>
+    var x: String =
+      for name in Array[String].values() do
+        name
+      else
+        "no names!"
+      end
+    env.out.print("x is " + s)
+```
```

Comment:
You added the new code block but did not remove the old code block. Will you make the edits so that your additions are incorporated with the example flow listed below?

That is the examples on this page proceed as:
1. Showing an if-else with return type `(String | Bool)`
2. Showing an if-else with return type `(String | None)`
3. Loop with literal Array[String] and no else block with return type `(String | None)`
4. Loop with literal (empty) Array[String] and no else block with return type `(String | None)`
5. Your loop with literal (empty) Array[String] and default else block with return type `String`

Some of the intervening text between examples will need to be rewritten to allow this flow -- for example, it would not longer make sense so say "And lastly, here x would be None." if that example (4 above) precedes your additional example (5 above).

Reply:
Not sure I was clear on this following proofreading the above comment, but I intend to mean "your loop" should replace the old problematic loop example as well as reorganize the examples to follow the above example flow. Your added code block should be followed up by the "Here the value of x is "no names!"" intervening text.